lib/typescript/render/providers/google/components/Suggestions/index.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import styles from './index.module.scss';
+import {DefaultMessageRenderingProps} from '../../../../components/index';
+import {
+  SuggestedReplies,
+  SuggestedActions,
+  AuthenticationRequestSuggestion,
+  LiveAgentRequestSuggestion,
+} from '../../googleModel';
+import {Image} from '../../../../components/Image';
+import {Text} from '../../../../components/Text';
+import linkIcon from 'assets/images/icons/link.svg';
+import phoneIcon from 'assets/images/icons/phone.svg';
+
+type SuggestionsRendererProps = DefaultMessageRenderingProps & {
+  text?: string;
+  fallback?: string;
+  image?: {
+    fileUrl: string;
+    altText: string;
+  };
+  suggestions:
+    | SuggestedReplies[]
```

Comment:
Maybe you could add a combined type in the `googleModel` ? that way you don't have to duplicate this in line 44-47 and in the model file itself?